### PR TITLE
Fix trash cart not available, showing console errors

### PR DESCRIPTION
**Pull request by wax100**
### What does it do?
Fix issue #14331 

### Related issue(s)/PR(s)
Fix issue #14331

### CODE REVIEW

**Comment by Jako** [comment]
Please use the right branch (3.x).

**Comment by JoshuaLuckers** [comment]
Thanks for taking time to submit this PR and help make MODX better. Since you are a first-time contributor; please sign the the [Contributor License Agreement (CLA)](https://docs.modx.com/community/contribute/becoming-a-contributor).

**Comment by wax100** [comment]
> Thanks for taking time to submit this PR and help make MODX better. Since you are a first-time contributor; please sign the the [Contributor License Agreement (CLA)](https://docs.modx.com/community/contribute/becoming-a-contributor).

I am already Logged In

**Review by Ibochkarev — COMMENTED** [review]
During the testing of the fix I get this error:
Dashboard | MODX Revolution 2019-03-28 10-10-05
Dashboard | MODX Revolution 2019-03-28 10-11-34

**Review by Ibochkarev — DISMISSED** [review]
Switching the system setting of `compress_js => no` - fixed the situation.

**Comment by JoshuaLuckers** [comment]
@wax100 Please sign it (again) using this URL: https://modx.com/community/contribute/cla/

**Comment by wax100** [comment]
> @wax100 Please sign it (again) using this URL: https://modx.com/community/contribute/cla/

Done!

File: manager/assets/modext/widgets/resource/modx.tree.resource.js [hunk 1]
```diff
@@ -235,8 +235,10 @@ Ext.extend(MODx.tree.Resource,MODx.tree.Tree,{
                         } else {
                             trashButton.tabEl.classList.add('active');
                         }
-                        trashButton.setTooltip(_('trash.manage_recycle_bin_tooltip', {count: data.object.deletedCount}));
-                    }
+                        trashButton.tooltip = new Ext.ToolTip({
+                            target: trashButton.tabEl,
+                            title: _('trash.manage_recycle_bin_tooltip', {count: data.object.deletedCount})
+                        });                    }
```
Comment (JoshuaLuckers) [inline manager/assets/modext/widgets/resource/modx.tree.resource.js]: Can you put this on a new line?

File: manager/assets/modext/widgets/resource/modx.tree.resource.js [hunk 2]
```diff
@@ -276,8 +278,10 @@ Ext.extend(MODx.tree.Resource,MODx.tree.Tree,{
...
```
Comment (JoshuaLuckers) [inline manager/assets/modext/widgets/resource/modx.tree.resource.js]: Same here.

File: manager/assets/modext/widgets/resource/modx.tree.resource.js (hunk 1 above)
Comment (wax100) [reply manager/assets/modext/widgets/resource/modx.tree.resource.js]: Done

File: manager/assets/modext/widgets/resource/modx.tree.resource.js (hunk 2 above)
Comment (wax100) [reply manager/assets/modext/widgets/resource/modx.tree.resource.js]: Done

**Review by JoshuaLuckers — APPROVED** [review]

**Review by Mark-H — APPROVED** [review]

**Pull request merged**